package com.ncc.JavaCore.TryCatchvsThrowException;

public class UncheckedExample {
//    public void convertToInt(String str) {
//        int num = Integer.parseInt(str); // không cần kiểm tra ngoại lệ tại thời điểm biên dịch
//        // ...
//    }

//    public static void main(String[] args) {
//        int num1=10;
//        int num2=0;
//        /*Since I'm dividing an integer with 0
//         * it should throw ArithmeticException
//         */
//        int res=num1/num2;
//        System.out.println(res);
//    }
//    Đoạn code trên khi chúng ta compile sẽ không gây bất kỳ lỗi
//    nào nhưng nếu chạy thì chúng ta sẽ bị ném ArithmeticException.
}
